src/circuits/joinsplit.tcc

```diff
@@ -399,6 +399,14 @@ public:
         }
     }
 
+    // Check the booleaness of packing variables
+    // Check the booleanees of phi and the a_sks
```

Comment:
booleaness

Reply:
Fixed.

Reply:
I commented on the line `// Check the booleanees of phi and the a_sks` and you changed all the others @rrtoledo